lua/tardis/interiors/default.lua

```diff
@@ -93,7 +93,7 @@ T.Interior = {
 		default_sonicdispenser     = {pos = Vector(-26.048, 42.31, 1.45),             ang = Angle(9.445, 46.482, -21.861),    },
 		default_helmic             = {pos = Vector(-26, -41, 4),                      ang = Angle(0, 330, 24.5),              },
 		default_sonic_inserted     = SonicModelExists() and
-	                                 {pos = Vector(-29.748, 40.71, -20.15),           ang = Angle(-100, -90, 90),             },
+									 {pos = Vector(-29.748, 40.71, -20.15),           ang = Angle(-100, -90, 90),             },
```

Comment:
nitpick: new alignment pushes the alignment out of the closing bracket

Reply:
GitHub shows the discrepancies between space and tabs when they are both used in the same file. Only reason it looks fine in your editor is the tab width is different.

You can see here these is a mixture of tabs and spaces even on the same line for most of these. I don't mind which we use but it should be consistent within the codebase ideally or at least the file

![image](https://user-images.githubusercontent.com/2363642/151672300-5a936075-54d7-4411-af78-6746b0227e9b.png)